### DISCUSSION TITLE
What does the final "R" in DCRMR mean to you today?

### DISCUSSION THREAD

**Discussion by ladylazarus3** [rbms-bsc/DCRMR#145] [General]
For me and in this particular moment, the final "R" in DCRMR is the ROAR of the autumn wind as it rustles through crispy and crunchy leaves.  How does the final "R" catch you right now?

**Comment by eblakedc** [comment]
![Be_Are](https://user-images.githubusercontent.com/107492238/190834721-9a088c6b-05b8-4ace-816a-7abdc9d85a29.jpg)

**Reply by ladylazarus3** [reply]
Aye.  The code is more what you'd call guidelines than actual rules. <3 <3 <3  
AAAAARRRR 🏴‍☠️
